src/i18n/strings/en_EN.json

```diff
@@ -3216,6 +3216,7 @@
             "blocked": "The sender has blocked you from receiving this message",
             "historical_event_no_key_backup": "Historical messages are not available on this device",
             "historical_event_unverified_device": "You need to verify this device for access to historical messages",
+            "historical_event_user_not_joined": "You don't have access to this message",
```

Comment:
Should this be a bit more specific? eg: "You don't have access to this message because it was sent before you joined this room"

Reply:
This was the wording that was decided on within the crypto team.  The rationale was that because it was a message that could be repeated often, we didn't want it to be too long.  I believe the plan is to eventually (along with the pre-login UTD) add a banner that can give more explanation.